### DISCUSSION TITLE
I can't download files from remote URLs with nhentai plugin.

### DISCUSSION THREAD

**Discussion by Qingshui001** [Difegue/LANraragi#804] [Q&A] [answered]
LRR Version and OS
Debian 11，docker :version 0.8.9.
***
I can't download files from remote URLs with nhentai plugin.
I have configured the UserAgent, csrftoken cookie and cf_clearance cookie, but I still encountered an error when using the nhentai plugin to download documents. I am not sure where the problem is.
***
Bug Details & Matching Logs
	 Error while processing archive.
(Can't resolve: Unrecognized service at /home/koyomi/lanraragi/script/../lib/LANraragi/Model/Upload.pm line 172
***
The content of the line 172 is " my $content_disp = $tx->result->headers->content_disposition; "

**Comment by Difegue** [comment, marked as answer]
👋 There's no nhentai plugin for downloading URLs currently, are you mistaking it for the nhentai metadata plugin perchance?  

NH doesn't offer a clean API to download zipped files, and since the LRR plugin system doesn't support scraping/page-by-page scenarios, I just don't support it as a download source for now.
I recommend using the chaika or EH downloaders instead.
